modules/admin/admin-ui/src/main/resources/web/admin/apps/content-studio/js/app/publish/ContentPublishDialog.ts

```diff
@@ -146,12 +146,12 @@ export class ContentPublishDialog extends ProgressBarDialog {
             return this.reloadPublishDependencies(childrenNotLoadedYet);
         } else {
             // apply the stash to avoid extra heavy request
-            this.lockControls();
+            this.updateSubTitleShowScheduleAndButtonCount();
             this.loadMask.show();
             setTimeout(() => {
                 this.setDependantItems(stashedItems.slice());
                 this.centerMyself();
-                this.unlockControls();
+                this.updateSubTitleShowScheduleAndButtonCount();
                 this.loadMask.hide();
```

Comment:
You are right. I replaced the first `this.updateSubTitleShowScheduleAndButtonCount()` call with simple `lockControls()` call. We need the second `this.updateSubTitleShowScheduleAndButtonCount()` call, though, since the dependent items are restored from stash and we should check, whether we need to enable the Publish. Also, we need to recalculate the number of items to be published.